todo_app/app.py

```diff
@@ -1,31 +1,35 @@
 from flask import Flask, render_template, redirect
 from flask.globals import request
 from todo_app.data.trello_items import get_item, add_item, get_items, save_item
-import sys
 
-from todo_app.flask_config import Config
+from todo_app.view_model import ViewModel
+from todo_app import flask_config
 
-app = Flask(__name__)
-app.config.from_object(Config())
+def create_app():
 
+    app = Flask(__name__)
+    app.config.from_object(flask_config.Config())
 
-@app.route('/')
-def index():
-    return render_template('index.html', items=get_items())
+    @app.route('/')
+    def index():
+        view_model = ViewModel(get_items())
+        return render_template('index.html', view_model=view_model)
 
+    @app.route('/item', methods=['POST', 'PUT'])
```

Comment:
Should just be `methods=['POST']` unless you are sending a put request to Flask via javascript.

And then you can get rid of the corresponding if statement in the function body